MediaBrowser.Model/Dlna/DirectPlayProfile.cs

```diff
@@ -31,7 +32,7 @@ public bool SupportsVideoCodec(string codec)
 
         public bool SupportsAudioCodec(string codec)
         {
-            return (Type == DlnaProfileType.Audio || Type == DlnaProfileType.Video) && ContainerProfile.ContainsContainer(AudioCodec, codec);
+            return (Type is DlnaProfileType.Audio or DlnaProfileType.Video) && ContainerProfile.ContainsContainer(AudioCodec, codec);
```

Comment:
Not a fan of this, I'd rather stay consistent with the current codebase.

Reply:
I can revert it if you want, it was suggested by Rider and I thought it'd make sense.